pandas/tests/frame/test_to_csv.py

```diff
@@ -575,6 +575,13 @@ def test_to_csv_headers(self):
             recons.reset_index(inplace=True)
             tm.assert_frame_equal(to_df, recons)
 
+    def test_to_csv_raises_on_header_prefix(self):
```

Comment:
Is this testing to_csv or read_csv? Looks like it's in the to_csv file, but is testing read_csv?

Reply:
It's testing read_csv that's a mistake in naming it.

But I actually tried putting this test in `pandas/tests/io/parser/test_common.py`. But the problem there is that all the tests use `io.parsers.read_csv` there. And `io.parsers.read_csv` doesn't seem to be going into `CParserWrapper`. So I tried putting it here. But now I am putting it back in `test_common.py` and will import the `pandas.read_csv` there.

Reply:
Can you take a look, please?